packages/components/src/internal/components/navigation/NavigationBar.tsx

```diff
@@ -92,7 +92,7 @@ export const NavigationBar: FC<Props> = memo(props => {
                 <nav className="navbar navbar-container test-loc-nav-header">
                     <div className="container">
                         <div className="row">
-                            <div className="navbar-left col-xs-8 col-md-7">
+                            <div className="navbar-left col-xs-8 col-md-6">
```

Comment:
Switching to `md-6` breaks the longest menu item use case (and possibly others). With #690 I tweaked these to ensure the layout didn't start to overlap when the Folder name is long. My suspicion is you changed these to account for a large number of notifications. We may have to do something more complex here to get a consistent layout. Here is the use case I'm describing:

With `col-md-6` at screen width 1135px:
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/3926239/171270678-4fb7b61e-3578-4376-ae60-2e0fd80877dc.png">

With `col-md-7` at screen width 1135px:
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/3926239/171270830-3887ea18-e5fc-4de9-a226-2969c5cc6b79.png">

Reply:
A smaller search input box seems reasonable.